projects/bp-gallery/cypress/e2e/bulk-edit/link-pictures-with-texts.cy.ts

```diff
@@ -0,0 +1,345 @@
+// ignore entire suite while #431 is unresolved
```

Comment:
wollt ihr das vllt in einen anderen branch packen, der dann zum PR wird wenn 431 durch ist? bzw. ihn einfach mit 431 hinzufügen?

Reply:
Okay :)